cg/meta/workflow/balsamic.py

```diff
@@ -34,9 +34,9 @@
 
 
 PANELS_WITH_LOQUSDB_DUMP_FILES_MAP: dict[str, str] = {
-    "GMSmyeloid": "loqusdb_cancer_somatic_myeloid_snv_variants_export-20250920-.vcf.gz",
-    "GMSlymphoid": "loqusdb_cancer_somatic_lymphoid_snv_variants_export-20250920-.vcf.gz",
-    "Twist Exome Comprehensive": "loqusdb_cancer_somatic_exome_snv_variants_export-20250920-.vcf.gz",
+    BalsamicObservationPanel.MYELOID: "loqusdb_cancer_somatic_myeloid_snv_variants_export-20250920-.vcf.gz",
+    BalsamicObservationPanel.LYMPHOID: "loqusdb_cancer_somatic_lymphoid_snv_variants_export-20250920-.vcf.gz",
+    BalsamicObservationPanel.EXOME: "loqusdb_cancer_somatic_exome_snv_variants_export-20250920-.vcf.gz",
```

Comment:
This is the wrong place to have this discussion. But Its better to have it here then forget about it. Shouldn't the paths to the panels be better kept in the cg config?